### ISSUE TITLE
[FEATURE_REQUEST] Logit bias support for text completion models (like Ooba, Aphrodite and Tabby)

### ISSUE THREAD

**Issue by DutchEllie** [SillyTavern/SillyTavern#1569]
### Have you searched for similar requests?

Yes

### Describe the solution you'd like

Chat completion backends have logit bias support in the settings panel, but text completion backends do not. Ooba, Tabby and Aphrodite all support logit bias in their API specs. Please add the ability to send logit bias to those as well when generating.

### Describe alternatives you've considered

Proxying OpenAI chat completion to one of those backends, but honestly you'd lose so many sampler options (like min p) that make it completely worthless even trying.

### Additional context

I was told that until sequential logit bias is not implemented anywhere, this is worthless to add. I don't believe it is though, as people report successes with logit bias for the backends that support it. Also, it should definitely work for single logits (like `.` or `"`)

### Priority

Medium (Would be very useful)

### Is this something you would be keen to implement?

No

**Comment by Cohee1207** [comment]
Correction: Tabby doesn't support logit bias at all. Ooba accepts it but fails at processing with internal error. Aphrodite wasn't tested as the latest version doesn't work on colab. Other compatible endpoints like llama.cpp should work.

**Comment by DutchEllie** [comment]
I can't believe I was lied to, deceived and bamboozled... The documentation for those tools surely indicated it would work, but apparently they just **don't**. Incredible

Thank you so much Cohee. At least we're ready for when the support does ever arrive.

**Comment by Cohee1207** [comment]
@bdashore3 crunched all night to get you logit bias in Tabby. Update and try it out.

**Comment by DutchEllie** [comment]
What the hell!? That's amazing! You guys are so fucking cool man! I should be less of a dick :bow: 

Thank you guys so much!
